lib/grid_search.py

```diff
@@ -76,6 +78,7 @@ def train(self):
             self.all_errors[current_key] = dict()
             self.all_errors[current_key]['train_recall'] = train_recall
             self.all_errors[current_key]['test_recall'] = test_recall
+        self.dump_csv(all_results)
```

Comment:
I know, but the verbose is meant to report what's happening generally in the execution. Also if someone new runs the code.